provider/test-programs/cloudrun-service/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as gcp from "@pulumi/gcp";
+
+new gcp.cloudrun.Service("my-service", {
```

Comment:
Can we also add a test for `google_cloud_run_domain_mapping`? That's also using PRC now

Reply:
Ah, nevermind. Just saw the remark in the description